src/index.ts

```diff
@@ -18,6 +18,7 @@ import * as Promise from 'bluebird';
 import { EventEmitter } from 'events';
 import * as http from 'http';
 import * as net from 'net';
+import * as pkg from 'pjson';
```

Comment:
You should be able to just do:
```suggestion
import { version } from '../package.json'
```
You'll need to add
```json
"resolveJsonModule": true,
```
to tsconfig.json as well though, and then you'll also get types tailored for this specific package.json